app/javascript/shared/constants/messages.js

```diff
@@ -0,0 +1,12 @@
+export const MESSAGE_STATUS = {
+  FAILED: 'failed',
+  SUCCESS: 'sent',
```

Comment:
SENT?

Reply:
Matching response from action cable

Reply:
I meant the label of sent, it is still 'success'.